src/tracing/errors.ts

```diff
@@ -25,6 +25,11 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
 
     // If the error is based on a user rejecting, it should not be considered an exception.
     if (didUserReject(error)) return null
+
+    // This is caused by HTML being returned for a chunk from Cloudflare.
+    // Usually, it's the result of a 499 exception right before it, which should be handled.
+    // Therefore, this can be ignored.
+    if (error.message.match(/Unexpected token '<'/)) return null
```

Comment:
This should be a SyntaxError - I think you should be as specific as possible here:

```suggestion
    if (error instanceof SyntaxError && error.message.match(/Unexpected token '<'/)) return null
```